src/server/handlers/launchpad.js

```diff
@@ -83,6 +83,14 @@ const RESPONSE_SNAPCRAFT_YAML_NO_NAME = {
   }
 };
 
+const RESPONSE_SNAP_NAME_NOT_REGISTERED = {
+  status: 'error',
+  payload: {
+    code: 'snap-name-not-registered',
+    message: 'Snap name is not registered in the store'
```

Comment:
It could wait until later, but would it be quick to change this to "The name provided in the snapcraft.yaml ($name) is not registered in the store. Please register it \<a href="https://myapps.developer.ubuntu.com/dev/click-apps/register-name/?name=$name">here</a> before trying again."